core/services/feeds/models.go

```diff
@@ -0,0 +1,87 @@
+package feeds
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+type ManagerService struct {
+	ID        int32
+	Name      string
+	URI       string
+	PublicKey PublicKey
+	JobTypes  pq.StringArray
+	Network   string
```

Comment:
What is network?

Reply:
Network is the supported blockchain. The terminology comes from the EngOps team

Reply:
Does it come with a ChainID? That would be handy since then we can start linking it up to data on our side.